### DISCUSSION TITLE
Customer specific language files ?

### DISCUSSION THREAD

**Discussion by SchunckMarkus** [WeblateOrg/weblate#7102] [Q&A]
Dear community,

I am not familiar with Weblate and I am just trying it for a day...

I am working on a product, which shall support a larger number of smaller customers/tenants. It happens from time to time, that some customers (tenants) are not satisfied with the translated resources. They are asking us to change some of the resources.
Since we are running a single hosted system, this means we need to provide a dedicated translation for such a customer.

So I wonder whether I can achieve this in Weblate.

I am thinking about having a directory struture like:
/i18n/eng/customer/ 

is there a way to achieve this ?

Or would I need to use a structure like:
/i18n/en_customer_a/
/i18n/en_customer_b/  ?

Could I configure Weblate to do the second option by changing the languages.csv file ?

Thanks
Markus

**Comment by nijel** [comment]
Language codes like `en_customer_a` will work just fine. Weblate will detect these as being English variants and everything will work as expected. This is how most people create customer-specific translations.
